modules/agar/src/main/ts/ephox/agar/assertions/ApproxComparisons.ts

```diff
@@ -22,7 +22,7 @@ const is = (target: string): CombinedAssert => {
 
   const strAssert = (label: TestLabel, actual: string) => {
     const c = compare(actual);
-    assertOnBool(c, TestLabel.concat(label, '\nExpected value: ' + target), actual);
+    assertOnBool(c, TestLabel.concat(label, '\nExpected value: ' + JSON.stringify(target)), actual);
```

Comment:
This was just something that annoyed me when running tests, as it printed something like
```
Expected value: width: 500px, Actual value: "width: 500px"
```
So just added this to normalize the label/report.